app/index.js

```diff
@@ -84,6 +83,10 @@ module.exports = class extends Generator {
     this.fs.writeJSON(this.destinationPath('package.json'), pkg);
   }
 
+  conflicts() {
+    this.fs.append(this.destinationPath('.eslintignore'), 'generators/*/templates\n');
+  }
```

Comment:
> By the way, we need to handle all generator valid structure. The nesting inside generators/ folder is not required.

Yeah right. Changed that into `**/templates` and did a `--amend` commit and pushed.